sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuPythonHelper.scala

```diff
@@ -95,6 +98,7 @@ object GpuPythonHelper extends Logging {
       pyF.envVars.put("RAPIDS_PYTHON_ENABLED", isPythonOnGpuEnabled.toString)
       pyF.envVars.put("RAPIDS_UVM_ENABLED", isPythonUvmEnabled)
       pyF.envVars.put("RAPIDS_POOLED_MEM_ENABLED", isPythonPooledMemEnabled)
+      pyF.envVars.put("RAPIDS_ARENA_MEM_ENABLED", isPythonArenaMemEnabled)
```

Comment:
I see this code setting this environment variable, but no corresponding code acting on it on the Python side (e.g.: python/rapids/worker.py).

Reply:
Removed.